include/golioth/location.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <golioth/client.h>
+#include <golioth/golioth_status.h>
+#include <stdint.h>
+#include <zcbor_encode.h>
+
+struct golioth_location_rsp
+{
+    /** Latitudal position in nanodegrees (0 to +-180E9) */
+    int64_t latitude;
+    /** Longitudal position in nanodegrees (0 to +-180E9) */
+    int64_t longitude;
+
+    /** Accuracy in meters */
+    int64_t accuracy;
+};
+
+#define GOLIOTH_LOCATION_FLAG_WIFI BIT(0)
```

Comment:
is `BIT()` in the C standard library or is this Zephyr-specific?

Reply:
Good catch! Hmmm, that is Zephyr/Linux kernel specific... but hey... it was built successfully for all platforms I think. Anyway, we can switch to `(1 << 0)` just in case.

Reply:
Well, it was not built, as there is only Zephyr example. So yeah, good point. Will stich to bit shift directly.